content/packages/working-with-a-github-packages-registry/working-with-the-rubygems-registry.md

```diff
@@ -63,7 +62,8 @@ For example, you would create or edit a *~/.gem/credentials* to include the foll
 :github: Bearer TOKEN
 ```
 
-To install gems, you need to authenticate to {% data variables.product.prodname_registry %} by editing the *~/.gemrc* file for your project to include `https://USERNAME:TOKEN@{% ifversion fpt or ghec %}rubygems.pkg.github.com{% else %}REGISTRY-URL{% endif %}/OWNER/`. You must replace:
+#### Installing Gems
```

Comment:
```suggestion
```

Removed the headers for now to be consistent with our current style. I do understand what you were trying to do there, though. I'll open a followup internal issue for us to take a look at the current iteration of the content and make sure the structure isn't impeding comprehension. Thanks!